Formula/coreos-ct.rb

```diff
@@ -0,0 +1,56 @@
+class CoreosCt < Formula
+  desc "Convert a Container Linux Config into Ignition"
+  homepage "https://coreos.com/os/docs/latest/configuration.html"
+  url "https://github.com/coreos/container-linux-config-transpiler/archive/v0.6.1.tar.gz"
+  sha256 "57cafa9db08caa6bed0290612bebf5d1a763a96fdb21f23de1681fd266b0c3c3"
+  head "https://github.com/coreos/container-linux-config-transpiler"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "all"
+    bin.install "./bin/ct"
+  end
+
+  test do
+    assert_equal "ct ", shell_output("#{bin}/ct -version").chomp
```

Comment:
probably means we need to use the :tag => + :revision => instead of url and sha256

Reply:
Maybe, though I feel it might be better to just remove that line -- it's not really testing the functionality of the program.  What do you think, though?

Reply:
The issue is not about whether this should be in the test block or not (I agree it should not) but whether the software is able to actually report its version correctly.

Reply:
Submitted an issue upstream.  :wink: